sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -362,6 +500,29 @@ public virtual Response PostLedgerEntry(RequestContent content, string subLedger
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="transactionId"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="transactionId"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <example>
+        /// This sample shows how to call GetLedgerEntryAsync with required parameters and properties.
```

Comment:
What are the properties mentioned here?  It looks like the method is being called with a single parameter.

Reply:
I've split the cases into finer granularity. Now:
- If there are both body parameters and non-body parameters, then we return `parameters and request content`.
- If there are only non-body parameters, then we return `parameters`.
- If there are only body parameters, then we return `request content`.